<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page_notes</class>
 <widget class="QMainWindow" name="Page_notes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notes</string>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget_Navigation">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="FolderNavigationWidget" name="tab_Notes">
        <attribute name="icon">
         <iconset resource="../Manager.qrc">
          <normaloff>:/fugue-icons/folder-tree</normaloff>:/fugue-icons/folder-tree</iconset>
        </attribute>
        <attribute name="title">
         <string>Notes</string>
        </attribute>
       </widget>
       <widget class="TagsNavigationWidget" name="tab_Tags">
        <attribute name="icon">
         <iconset resource="../Manager.qrc">
          <normaloff>:/fugue-icons/tag-label</normaloff>:/fugue-icons/tag-label</iconset>
        </attribute>
        <attribute name="title">
         <string>Tags</string>
        </attribute>
       </widget>
       <widget class="DateNavigationWidget" name="tab_Dates">
        <attribute name="icon">
         <iconset resource="../Manager.qrc">
          <normaloff>:/fugue-icons/calendar-blue</normaloff>:/fugue-icons/calendar-blue</iconset>
        </attribute>
        <attribute name="title">
         <string>Dates</string>
        </attribute>
       </widget>
      </widget>
      <widget class="NotesTabWidget" name="tabWidget_EditNotes">
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBarManage">
   <property name="windowTitle">
    <string>Manage</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddNote"/>
   <addaction name="actionAddNoteFromClipboard"/>
   <addaction name="actionAddNoteFromScreen"/>
   <addaction name="actionAddFolder"/>
   <addaction name="separator"/>
   <addaction name="actionDelete"/>
   <addaction name="actionClearTrash"/>
  </widget>
  <widget class="QToolBar" name="toolBarNote">
   <property name="windowTitle">
    <string>Note</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSaveNote"/>
   <addaction name="actionSaveNoteAs"/>
   <addaction name="separator"/>
   <addaction name="actionShowNote"/>
   <addaction name="actionHideNote"/>
   <addaction name="separator"/>
   <addaction name="actionShowAllNotes"/>
   <addaction name="actionHideAllNotes"/>
   <addaction name="separator"/>
   <addaction name="actionSaveAllNotes"/>
   <addaction name="actionTopNote"/>
   <addaction name="separator"/>
   <addaction name="actionPrintNote"/>
   <addaction name="actionPreviewPrintNote"/>
  </widget>
  <action name="actionAddNote">
   <property name="icon">
    <iconset resource="../Manager.qrc">
     <normaloff>:/fugue-icons/plus</normaloff>:/fugue-icons/plus</iconset>
   </property>
   <property name="text">
    <string>Add Note</string>
   </property>
   <property name="statusTip">
    <string>Add Note</string>
   </property>
   <property name="whatsThis">
    <string>Add Note</string>
   </property>
  </action>
  <action name="actionAddNoteFromClipboard">
   <property name="icon">
    <iconset resource="../Manager.qrc">
     <normaloff>:/Manager/add-from_clipboard</normaloff>:/Manager/add-from_clipboard</iconset>
   </property>
   <property name="text">
    <string>Add a Note From The Clipboard</string>
   </property>
   <property name="statusTip">
    <string>Add a Note From The Clipboard</string>
   </property>
   <property name="whatsThis">
    <string>Add a Note From The Clipboard</string>
   </property>
  </action>
  <action name="actionSaveNote">
   <property name="icon">
    <iconset resource="../../Note/Note.qrc">
     <normaloff>:/Note/save</normaloff>:/Note/save</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Save</string>
   </property>
   <property name="whatsThis">
    <string>Save</string>
   </property>
  </action>
  <action name="actionShowNote">
   <property name="icon">
    <iconset resource="../Manager.qrc">
     <normaloff>:/Manager/show</normaloff>:/Manager/show</iconset>
   </property>
   <property name="text">
    <string>Show Note</string>
   </property>
   <property name="statusTip">
    <string>Show Note</string>
   </property>
   <property name="whatsThis">
    <string>Show Note</string>
   </property>
  </action>
  <action name="actionHideNote">
   <property name="icon">
    <iconset resource="../Manager.qrc">
     <normaloff>:/Manager/hide</normaloff>:/Manager/hide</iconset>
   </property>
   <property name="text">
    <string>Hide Note</string>
   </property>
   <property name="statusTip">
    <string>Hide Note</string>
   </property>
   <property name="whatsThis">
    <string>Hide Note</string>
   </property>
  </action>
  <action name="actionShowAllNotes">
   <property name="icon">
    <iconset resource="../Manager.qrc">
     <normaloff>:/Manager/show_all</normaloff>:/Manager/show_all</iconset>
   </property>
   <property name="text">
    <string>Show All Notes</string>
   </property>
   <property name="statusTip">
    <string>Show All Notes</string>
   </property>
   <property name="whatsThis">
    <string>Show All Notes</string>
   </property>
  </action>
  <action name="actionHideAllNotes">
   <property name="icon">
    <iconset resource="../Manager.qrc">
     <normaloff>:/Manager/hide_all</normaloff>:/Manager/hide_all</iconset>
   </property>
   <property name="text">
    <string>Hide All Notes</string>
   </property>
   <property name="statusTip">
    <string>Hide All Notes</string>
   </property>
   <property name="whatsThis">
    <string>Hide All Notes</string>
   </property>
  </action>
  <action name="actionTopNote">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../Note/Note.qrc">
     <normaloff>:/Note/tacks</normaloff>:/Note/tacks</iconset>
   </property>
   <property name="text">
    <string>On top of all windows</string>
   </property>
   <property name="statusTip">
    <string>On top of all windows</string>
   </property>
   <property name="whatsThis">
    <string>On top of all windows</string>
   </property>
  </action>
  <action name="actionPrintNote">
   <property name="icon">
    <iconset resource="../../Note/Note.qrc">
     <normaloff>:/Note/print</normaloff>:/Note/print</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="statusTip">
    <string>Print</string>
   </property>
   <property name="whatsThis">
    <string>Print</string>
   </property>
  </action>
  <action name="actionPreviewPrintNote">
   <property name="icon">
    <iconset resource="../../Note/Note.qrc">
     <normaloff>:/Note/preview-print</normaloff>:/Note/preview-print</iconset>
   </property>
   <property name="text">
    <string>Preview Print</string>
   </property>
   <property name="statusTip">
    <string>Preview Print</string>
   </property>
   <property name="whatsThis">
    <string>Preview Print</string>
   </property>
  </action>
  <action name="actionSaveAllNotes">
   <property name="icon">
    <iconset resource="../Manager.qrc">
     <normaloff>:/Manager/save_all</normaloff>:/Manager/save_all</iconset>
   </property>
   <property name="text">
    <string>Save All Notes</string>
   </property>
   <property name="statusTip">
    <string>Save All Notes</string>
   </property>
   <property name="whatsThis">
    <string>Save All Notes</string>
   </property>
  </action>
  <action name="actionAddNoteFromScreen">
   <property name="icon">
    <iconset resource="../Manager.qrc">
     <normaloff>:/Manager/screenshot</normaloff>:/Manager/screenshot</iconset>
   </property>
   <property name="text">
    <string>Add a Note From Screen</string>
   </property>
   <property name="statusTip">
    <string>Add a Note From Screen</string>
   </property>
   <property name="whatsThis">
    <string>Add a Note From Screen</string>
   </property>
  </action>
  <action name="actionAddFolder">
   <property name="icon">
    <iconset resource="../Manager.qrc">
     <normaloff>:/fugue-icons/folder--plus</normaloff>:/fugue-icons/folder--plus</iconset>
   </property>
   <property name="text">
    <string>Add Folder</string>
   </property>
   <property name="statusTip">
    <string>Add Folder</string>
   </property>
   <property name="whatsThis">
    <string>Add Folder</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../Manager.qrc">
     <normaloff>:/fugue-icons/cross</normaloff>:/fugue-icons/cross</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="statusTip">
    <string>Delete</string>
   </property>
   <property name="whatsThis">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionClearTrash">
   <property name="icon">
    <iconset resource="../Manager.qrc">
     <normaloff>:/Manager/remove_all</normaloff>:/Manager/remove_all</iconset>
   </property>
   <property name="text">
    <string>Clear Trash</string>
   </property>
   <property name="statusTip">
    <string>Clear Trash</string>
   </property>
   <property name="whatsThis">
    <string>Clear Trash</string>
   </property>
  </action>
  <action name="actionSaveNoteAs">
   <property name="icon">
    <iconset resource="../../Note/Note.qrc">
     <normaloff>:/Note/save-as</normaloff>:/Note/save-as</iconset>
   </property>
   <property name="text">
    <string>Save Note As</string>
   </property>
   <property name="statusTip">
    <string>Save Note As</string>
   </property>
   <property name="whatsThis">
    <string>Save Note As</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FolderNavigationWidget</class>
   <extends>QWidget</extends>
   <header>NavigationPanel/foldernavigationwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DateNavigationWidget</class>
   <extends>QWidget</extends>
   <header>NavigationPanel\datenavigationwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TagsNavigationWidget</class>
   <extends>QWidget</extends>
   <header>NavigationPanel/tagsnavigationwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NotesTabWidget</class>
   <extends>QTabWidget</extends>
   <header>NavigationPanel/notestabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Note/Note.qrc"/>
  <include location="../Manager.qrc"/>
 </resources>
 <connections/>
</ui>
